nibabel/testing/__init__.py

```diff
@@ -164,3 +166,14 @@ def __init__(self, *args, **kwargs):
         warnings.warn('catch_warn_reset is deprecated and will be removed in '
                       'nibabel v3.0; use nibabel.testing.clear_and_catch_warnings.',
                       FutureWarning)
+
+
+EXTRA_SET = os.environ.get('NIPY_EXTRA_TESTS', '').split(',')
+
+
+def runif_extra_has(test_str):
+    """Decorator checks to see if NIPY_EXTRA_TESTS env var contains test_str"""
+    def decorator(func):
+        return skipif(test_str not in EXTRA_SET,
+                      "Skip {0} tests.".format(test_str))(func)
+    return decorator
```

Comment:
Sorry, I'm going to be annoying and insist on removing the `decorator` function. What you have is:

    def f(x):
        return g(x)
    return f

When you could just have `return g`. Just use:

    def runif_extra_has(test_str):
        return skipif(test_str not in EXTRA_SET, "Skip {0} tests.".format(test_str))